docs/getting-started.md

```diff
@@ -21,11 +21,44 @@ or locally see [Installing Gravitino playground locally](#installing-gravitino-p
 
 If you are using AWS and want to access the instance remotely, be sure to read [Accessing Gravitino on AWS externally](#accessing-gravitino-on-aws-externally)
 
+### Key Sections:
+
+1. **Installation Methods:**
+   - Explore different installation methods, from using Docker to setting up Gravitino on cloud platforms or locally.
+
+2. **Java Development Kit (JDK):**
```

Comment:
We follow the Google Developer Guide for documents and that uses sentence case, so we'll need to remove the capitalisation here and elsewhere.